plugin/jvm/src/main/java/com/twosigma/beaker/easyform/EasyForm.java

```
@@ -271,32 +272,48 @@ public String getCaption() {
     return caption;
   }
 
-  private HashMap<String, String> getValuesMap() {
+  private HashMap<String, Object> getValuesMap() {
     return this.mapInstance;
   }
 
+  @Override
   public String get(final Object key) {
     checkComponentExists((String) key);
     return getComponentMap().get(key).getValue();
   }
 
-  public String put(final String key, final String value) {
+  @Override
+  public String put(final String key, final Object value) {
+    String _value = processValue(value);
     checkComponentExists(key);
     EasyFormComponent component = getComponentMap().get(key);
-    if (!component.checkValue(value)) {
+    if (!component.checkValue(_value)) {
```

should only allow strings and date objects, other objects should throw this exception.  eg this code should throw:

f = new EasyForm()
f.addDatePicker("d")
f["d"] = new ArrayList<Integer>()
f